tools/tpm2_rsaencrypt.c

```diff
@@ -131,24 +121,35 @@ static bool on_option(char key, char *value) {
     return true;
 }
 
+static bool on_args(int argc, char **argv) {
+
+    if (argc > 1) {
+        LOG_ERR("Only supports one hash input file, got: %d", argc);
+        return false;
+    }
+
+    ctx.input_path = argv[1];
+
+    return true;
+}
+
 bool tpm2_tool_onstart(tpm2_options **opts) {
 
     static const struct option topts[] = {
       {"key-handle",  required_argument, NULL, 'k'},
-      {"in-file",     required_argument, NULL, 'I'},
       {"out-file",    required_argument, NULL, 'o'},
       {"key-context", required_argument, NULL, 'c'},
     };
 
-    *opts = tpm2_options_new("k:I:o:c:", ARRAY_LEN(topts), topts,
-            on_option, NULL);
+    *opts = tpm2_options_new("k:o:c:", ARRAY_LEN(topts), topts,
+            on_option, on_args);
 
     return *opts != NULL;
 }
 
 static bool init(TSS2_SYS_CONTEXT *sapi_context) {
 
-    if (!((ctx.flags.k || ctx.flags.c) && ctx.flags.I)) {
+    if (!(ctx.flags.k || ctx.flags.c)) {
         LOG_ERR("Expected options I and (k or c)");
```

Comment:
You forgot to remove the `I` option from the error log printout.

Reply:
Done